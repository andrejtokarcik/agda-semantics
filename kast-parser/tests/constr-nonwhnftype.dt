data Test : Set 0 where   // FIXME je urcite zo Setu s vyssim indexom
  Constr : (Lambda x (Set 2) x) (Pi x (Set 1) Test)

; expr = Constr (Set 0)
