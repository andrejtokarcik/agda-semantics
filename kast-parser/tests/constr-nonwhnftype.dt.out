<T>
    <k>
        .K
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <tenv>
        expr |-> Test
    </tenv>
    <data>
        Constr |-> 'Pi___(x,, Set 1,, Test)
        Test |-> Set 0
    </data>
    <fun>
        expr |->  Constr Set 0 
    </fun>
    <meta>
        .Map
    </meta>
</T>
